cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/nick/network-sim/mptcp/arch/sim/test/buildtop/source/ns-3-dce/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/home/nick/network-sim/mptcp/arch/sim/test/buildtop/source/ns-3-dce/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 47348227341840: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-g\', \'-I/usr/include/python2.6\', \'-DNS3_LOG_ENABLE\', \'-DNS3_ASSERT_ENABLE\', \'-DHAVE_NS3_CORE=1\', \'-DHAVE_NS3_NETWORK=1\', \'-DHAVE_NS3_INTERNET=1\', \'-DHAVE_NS3_POINT_TO_POINT=1\', \'-DHAVE_NS3_TAP_BRIDGE=1\', \'-DHAVE_NS3_NETANIM=1\', \'-DHAVE_NS3_WIFI=1\', \'-DHAVE_NS3_CSMA=1\', \'-DHAVE_NS3_MOBILITY=1\', \'-DHAVE_NS3_POINT_TO_POINT_LAYOUT=1\', \'-DHAVE_NS3_MPI=1\', \'-DHAVE_NS3_LTE=1\', \'-DHAVE_NS3_APPLICATIONS=1\', \'-DHAVE_NS3_FD_NET_DEVICE=1\', \'-DHAVE_STDINT_H=1\', \'-DHAVE_INTTYPES_H=1\', \'-DHAVE_SYS_TYPES_H=1\', \'-DHAVE_SYS_STAT_H=1\', \'-DHAVE_DIRENT_H=1\', \'-DHAVE_GETCPUFEATURES=1\', \'-DHAVE_SIM_H=1\', \'-DKERNEL_STACK=Y\', \'-DHAVE_NS3_TOPOLOGY_READ=1\', \'-DHAVE_NS3_INTERNET_APPS=1\', \'-DHAVE_NS3_FLOW_MONITOR=1\', \'-DHAVE_HOOK_MANAGER_H=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
